addon/components/frost-modal-dialog.js

```diff
@@ -1,14 +1,32 @@
+/* global Ps */
 import Ember from 'ember'
-import layout from '../templates/components/frost-modal-dialog'
 import computed from 'ember-computed-decorators'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
 
 const {Component, assert} = Ember
 
-export default Component.extend({
-  layout: layout,
+export default Component.extend(PropTypeMixin, {
```

Comment:
@sandersky Can you elaborate on the issues?  I just added it to `ember-frost-core` using the implicit syntax